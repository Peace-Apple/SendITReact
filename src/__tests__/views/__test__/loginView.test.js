import React from "react";
import { shallow } from "enzyme";
import LoginView from "../../../views/loginView";

it("renders without crashing", () => {
  const wrapper = shallow(<LoginView />);
  expect(wrapper).toMatchSnapshot();
});
