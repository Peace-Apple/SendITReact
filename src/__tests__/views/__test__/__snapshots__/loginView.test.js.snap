// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`user login view should render without crashing 1`] = `
LoginView {
  "context": Object {},
  "handleSubmit": [Function],
  "onChange": [Function],
  "props": Object {},
  "refs": Object {},
  "state": Object {
    "errors": Object {},
    "password": "",
    "user_name": "",
  },
  "updater": Updater {
    "_callbacks": Array [],
    "_renderer": ReactShallowRenderer {
      "_context": Object {},
      "_didScheduleRenderPhaseUpdate": false,
      "_dispatcher": Object {
        "readContext": [Function],
        "useCallback": [Function],
        "useContext": [Function],
        "useDebugValue": [Function],
        "useEffect": [Function],
        "useImperativeHandle": [Function],
        "useLayoutEffect": [Function],
        "useMemo": [Function],
        "useReducer": [Function],
        "useRef": [Function],
        "useState": [Function],
      },
      "_element": <LoginView />,
      "_firstWorkInProgressHook": null,
      "_forcedUpdate": false,
      "_instance": [Circular],
      "_isReRender": false,
      "_newState": null,
      "_numberOfReRenders": 0,
      "_renderPhaseUpdates": null,
      "_rendered": <div>
        <LoginForm
          onChange={[Function]}
          onSubmit={[Function]}
        />
      </div>,
      "_rendering": false,
      "_updater": [Circular],
      "_workInProgressHook": null,
    },
  },
}
`;
